powerauth-java-http/src/main/java/io/getlime/security/powerauth/http/validator/ValueTypeValidator.java

```diff
@@ -127,11 +153,21 @@ public static boolean isValidSignatureValue(String signature) {
      * @return True if provided string is decimal and has expected length range.
      */
     public static boolean isDecimalString(String decimalString, int from, int to) {
-        if (decimalString != null && decimalString.matches("^[0-9]*$")) {
+        if (decimalString != null && decimalString.matches(decimalStringRegex)) {
             return decimalString.length() >= from && decimalString.length() <= to;
         } else {
             return false;
         }
     }
 
+    /**
+     * Check if the provided version is a valid one.
+     * @param version Version to check.
+     * @return True if provided version is valid, false otherwise.
+     */
+    @SuppressWarnings("BooleanMethodIsAlwaysInverted")
```

Comment:
Inverted logic means that you only use the method in the context where you check for inverted status. Example:

```java
public boolean isOk() {
   return this.checkOk();
}

// ... later in the code

if (!isOK) {
   // ... do something
}
```

The compiler suggests implementing the method `isNotOk` so that you can avoid the inversion in `if` statement, but this does not usually improve readability.

Reply:
`isNotOk` is fine for me, but I do not insist on it.